echo_handlers.go

```diff
@@ -73,14 +61,15 @@ func (svc *Service) RegisterSharedRoutes(e *echo.Echo) {
 	assetSubdir, _ := fs.Sub(embeddedAssets, "public")
 	assetHandler := http.FileServer(http.FS(assetSubdir))
 	e.GET("/public/*", echo.WrapHandler(http.StripPrefix("/public/", assetHandler)))
-	e.GET("/apps", svc.AppsListHandler)
-	e.GET("/apps/new", svc.AppsNewHandler)
-	e.GET("/apps/:pubkey", svc.AppsShowHandler)
-	e.POST("/apps", svc.AppsCreateHandler)
-	e.POST("/apps/delete/:pubkey", svc.AppsDeleteHandler)
+	e.GET("/api/getCSRFToken", svc.CSRFHandler)
+	e.GET("/api/apps", svc.AppsListHandler)
+	e.GET("/api/apps/:pubkey", svc.AppsShowHandler)
+	e.POST("/api/apps", svc.AppsCreateHandler)
+	e.POST("/api/apps/delete/:pubkey", svc.AppsDeleteHandler)
```

Comment:
should this be `e.DELETE("/api/apps/:pubkey")`?

Reply:
We need to pass the (anti-)csrf token, hence `POST`

Reply:
Figured it out, fixed now